tests/protos/kitchen-sink.proto

```diff
@@ -7,6 +7,7 @@
 package some_package;
 
 import "tests/protos/options.proto";
+import "google/protobuf/descriptor.proto";
```

Comment:
(see this pr, in particular tess's last comment https://github.com/dpup/pbnj/pull/10/files)